### PR TITLE
Feature/27 express data prep

### LINKED ISSUE #27: PSP Wizard
Commercial editions project newly contains a "Express Data Presp Wizard (aka PSP Wizard)". It should be documented for the users. 

Expected content: 
* Context. 
* Configuration. 
* Datastores. 
* Input & output. 
* UseCase walkthrough.

### PR DESCRIPTION
**Pull request by jhorcicka**
Resolves #27.  

Express Data Prep Wizard (aka PSP Wizard).

### CODE REVIEW

File: src/docbkx/chapter77-express-data-prep.xml
Comment (ClaudiaPHI) [inline src/docbkx/chapter77-express-data-prep.xml]: "the cleansed data, the invalid-data, the PDF"
needs an articol. "a/the/your financial summary"

File: src/docbkx/chapter77-express-data-prep.xml
Comment (ClaudiaPHI) [inline src/docbkx/chapter77-express-data-prep.xml]: We support other files as well, example "orderdb". But the data needs to registered as a datastore first.

Comment (ClaudiaPHI) [reply src/docbkx/chapter77-express-data-prep.xml]: Excel and Csv are required if the user wants to use the hot folder functionality

File: src/docbkx/chapter77-express-data-prep.xml
Comment (ClaudiaPHI) [inline src/docbkx/chapter77-express-data-prep.xml]: quotes instead of parenthesis

File: src/docbkx/chapter77-express-data-prep.xml
Comment (ClaudiaPHI) [inline src/docbkx/chapter77-express-data-prep.xml]: "The results"

File: src/docbkx/chapter77-express-data-prep.xml
Comment (ClaudiaPHI) [inline src/docbkx/chapter77-express-data-prep.xml]: use positive. delete "do not forget"

File: src/docbkx/chapter77-express-data-prep.xml
Comment (ClaudiaPHI) [inline src/docbkx/chapter77-express-data-prep.xml]: "This option is enable when you have selected the required extensions"

File: src/docbkx/chapter77-express-data-prep.xml
Comment (ClaudiaPHI) [inline src/docbkx/chapter77-express-data-prep.xml]: Do we have documentation for hot folder?  If not should we add it ? @LosD, @arjan ? a link might be useful

File: src/docbkx/chapter77-express-data-prep.xml
Comment (ClaudiaPHI) [inline src/docbkx/chapter77-express-data-prep.xml]: Don't we confuse the user with no real data ?

File: src/docbkx/chapter77-express-data-prep.xml
Comment (ClaudiaPHI) [inline src/docbkx/chapter77-express-data-prep.xml:217]: I think 'mapping' is more correct instead of header

File: src/docbkx/chapter77-express-data-prep.xml
Comment (ClaudiaPHI) [inline src/docbkx/chapter77-express-data-prep.xml:253]: need another  screenshot
image

**Comment by ClaudiaPHI** [comment]
I would also add what services are offered by the wizard: 
- Address Correction
- Sorting
- Email correction
- Phone correction

File: src/docbkx/chapter77-express-data-prep.xml
Comment (jhorcicka) [reply src/docbkx/chapter77-express-data-prep.xml]: Is 
```
Among other things you can find final cleansed and invalid data and a PDF report containing financial summary there.
``` 
Is this better?

File: src/docbkx/chapter77-express-data-prep.xml
Comment (LosD) [reply src/docbkx/chapter77-express-data-prep.xml]: I think we have a little bit, but that it's pretty limited.

File: src/docbkx/chapter77-express-data-prep.xml
Comment (jhorcicka) [reply src/docbkx/chapter77-express-data-prep.xml]: Please check "Datastores/edp_wizard_datastores" section again.

**Comment by LosD** [comment]
Many of the screenshot does not need to be so large.:
- When you instruct to click the wizard just show the single panel. No need for entire welcome screen.
image

- When you're uploading, only show the upload dialog.
image

- For the dropdown, only show the actual dropdown
image

I feel somewhat the same for the wizard pages, though it's more okay in that instance. But in general, there's _way_ too many screenshots for my liking.

File: src/docbkx/chapter77-express-data-prep.xml
Comment (jhorcicka) [reply src/docbkx/chapter77-express-data-prep.xml]: To be able to use the "quotes" I also changed "This" to "The". 

In my opinion the previous sentence was fine but I don't have problem changing it this way.

File: src/docbkx/chapter77-express-data-prep.xml
Comment (jhorcicka) [reply src/docbkx/chapter77-express-data-prep.xml]: I believe plurals do not require an article. But it is not wrong to add it, I guess. So be it. :-)

File: src/docbkx/chapter77-express-data-prep.xml
Comment (jhorcicka) [reply src/docbkx/chapter77-express-data-prep.xml]: Changed to 
```
During a typical DataCleaner Enterprise installation remember to check...
```

File: src/docbkx/chapter77-express-data-prep.xml
Comment (jhorcicka) [reply src/docbkx/chapter77-express-data-prep.xml]: Changed to 
```
This option is enabled only after you have selected Name and Address correction (on premise) item. 
```

File: src/docbkx/chapter77-express-data-prep.xml
Comment (jhorcicka) [reply src/docbkx/chapter77-express-data-prep.xml]: I added a link to: https://datacleaner.org/resources/docs/5.1/html_single/#monitor_scheduling_jobs

File: src/docbkx/chapter77-express-data-prep.xml
Comment (jhorcicka) [reply src/docbkx/chapter77-express-data-prep.xml]: Well, that depends. I can imagine a scenario where no "real data" is yet available. (Let's say the whole environment is being setup. )

With this piece of information an administrator does not have to try to get such data file. I can put it to brackets if it's better. 

If you don't agree, I'll remove it.

File: src/docbkx/chapter77-express-data-prep.xml
Comment (jhorcicka) [reply src/docbkx/chapter77-express-data-prep.xml:217]: I disagree. Mapping, in my opinion, is a morphism/function from a set of input columns to a set of meanings. 

In this case we are talking only about the input columns set. I can replace "header" with "number of columns and their names" if it's better.

File: src/docbkx/chapter77-express-data-prep.xml
Comment (jhorcicka) [reply src/docbkx/chapter77-express-data-prep.xml:253]: I don't understand. 

Instead of: 


You want: 


?

**Comment by jhorcicka** [comment]
@ClaudiaPHI: Good idea about provided services. I'll add a list to the "Context" section.

**Comment by jhorcicka** [comment]
@LosD: I'll reduce the sizes but I'd rather keep all of them. If there is someone else who disagrees then I will resign. :-)

File: src/docbkx/chapter77-express-data-prep.xml
Comment (ClaudiaPHI) [reply src/docbkx/chapter77-express-data-prep.xml:253]: first row is cut(addressline 1 seems wrong). It doesn't look nice

**Comment by arjansh** [comment]
Rename psp-wizard.properties to edp-wizard.properties. Also add a reference to the edp-wizard.properties provided by the installer to the documentation (stating you can copy it to your .datacleaner/{version}/ folder in your user home).

**Comment by jhorcicka** [comment]
1) "psp" -> "edp"
2) note about manual copying of the properties file

properties

@ClaudiaPHI: ready for re-review.

**Comment by LosD** [comment]
The plan is also to change `psp.outgoing.output.file.path` to `edp.outgoing.folder`.

File: src/docbkx/chapter77-express-data-prep.xml
Comment (ClaudiaPHI) [inline src/docbkx/chapter77-express-data-prep.xml]: The property has been renamed to 'edp.outgoing.folder' @jhorcicka

**Comment by jhorcicka** [comment]
Thanks for noticing @LosD, @ClaudiaPHI, changed.

**Comment by LosD** [comment]
LGTM!

**Review by LosD — APPROVED** [review]
LGTM

**Comment by LosD** [comment]
I'll let @ClaudiaPHI also have a say before merging, as she also had comments

File: src/docbkx/chapter77-express-data-prep.xml
Comment (ClaudiaPHI) [reply src/docbkx/chapter77-express-data-prep.xml:253]: @jhorcicka

File: src/docbkx/chapter77-express-data-prep.xml
Comment (ClaudiaPHI) [reply src/docbkx/chapter77-express-data-prep.xml:217]: It can have other names as long it has the same type and the same number of columns as far as I remember.

**Comment by ClaudiaPHI** [comment]
I have the comment about the screenshot. Otherwise LGTM!

File: src/docbkx/chapter77-express-data-prep.xml
Comment (jhorcicka) [reply src/docbkx/chapter77-express-data-prep.xml:253]: Ah, now I see what you meant. Updated.

**Pull request merged**
